publication-commons/src/main/java/no/unit/nva/publication/utils/RdfUtils.java

```diff
@@ -41,7 +41,7 @@ public static URI getTopLevelOrgUri(UriRetriever uriRetriever, URI id) {
         var data = attempt(() -> uriRetriever.getRawContent(id, APPLICATION_JSON)).orElseThrow();
 
         if (data.isEmpty()) {
-            return id;
+            return null;
```

Comment:
Now wondering what the thinking for the previous iteration was…

Reply:
yes, me too. Maybe to make the unit tests easier? You could maybe argue that it's not completely wrong. My current level is the "most top" unit that I know of. :-P

Reply:
could be that an Exception would be better than null here also